internal/util/grpcclient/client.go

```diff
@@ -11,14 +11,20 @@ import (
 	grpcopentracing "github.com/grpc-ecosystem/go-grpc-middleware/tracing/opentracing"
 	"github.com/milvus-io/milvus/internal/log"
 	"github.com/milvus-io/milvus/internal/util/funcutil"
-	"github.com/milvus-io/milvus/internal/util/retry"
 	"github.com/milvus-io/milvus/internal/util/trace"
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/backoff"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/keepalive"
 )
 
+const (
+	dialTimeout      = 5 * time.Second
+	keepAliveTime    = 10 * time.Second
```

Comment:
can we support configuring these variables?

Reply:
> can we support configuring these variables?

I think the timeout params do not need to be configuable. They should be set to constant variable.